packages/marathon/buildinfo.json

```diff
@@ -2,7 +2,7 @@
   "requires": ["java"],
   "single_source": {
     "kind": "url_extract",
-    "url": "http://downloads.mesosphere.io/marathon/v1.3.2/marathon-1.3.2.tgz",
+    "url": "https://downloads.mesosphere.io/marathon/v1.3.2/marathon-1.3.2.tgz",
```

Comment:
nit: can we change this to downloads.mesosphere.com as well? (ideally shouldn't use downloads.mesosphere.io anywhere).